app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewModel.kt

```diff
@@ -195,10 +215,16 @@ class AttendeeViewModel(
         }
         val eventId = event.value?.id
         if (eventId != null) {
-            val order = Order(
-                getId(), paymentMode, countryForOrder, "pending", amount,
-                attendees = attendeeList, event = EventId(eventId)
-            )
+            val order =
+                if (billingEnabled)
+                    Order(getId(), paymentMode, status = "pending", amount = amount,
```

Comment:
I guess it would be better if you use a constant for "pending" to avoid mistakes in future